config.toml

```diff
@@ -36,6 +36,11 @@ sphinxopts = [
     '-D latex_elements.fontenc=\\usepackage{fontspec}',
 ]
 
+[languages.bn_IN]
```

Comment:
Isn't Bengali in Bangladesh very similar (practically the same) as Bengali in India? Could we cover both regions with your translation project?